src/app/modules/group/components/group-invite-users/group-invite-users.component.ts

```diff
@@ -115,7 +122,7 @@ export class GroupInviteUsersComponent {
         this.displayResponse(res);
 
         // Clear the textarea
-        if (this.textArea) this.textArea.setValue('');
+        this.inviteForm.get(this.inputName)?.patchValue('');
```

Comment:
Use the var discussed above.

`setValue` looks safer than patchValue when the goal is not to update the whole form (as we do in edit). You can also decide to emit the event (in both fct) but probably here it is not what we want.

Reply:
Isn't patchValue what we want here ? https://www.tektutorialshub.com/angular/setvalue-patchvalue-in-angular/#setvalue-vs-patchvalue.

Reply:
IMO, `patchValue` is more when we work a big object with many attributes and we want to replace one. Here we want to reset one specific known attribute, we prefer if it properly crash if it does not work (instead of ignoring as patchValue would do).

Reply:
Okay, I've changed it